### ISSUE TITLE
No explainations for total noobs

### ISSUE THREAD

**Issue by pirate486743186** [EasyRPG/easyrpg.org#41]
I'm a total noob. I have no idea what's rpg maker. Where do i get games to run with easyrpg? And what's RTP? How i put all this together?

The site and main page here, should mention rpgmaker.net . Specifically the links for 2000/2003 and easy rpg lists of games.

also explain, that in general you need RTP, and provide the links(yes, i found the FAQ). Also, a caveat about this, is that some games actually want the japanese RTP.

I had to go hunt for answers all over the internet....

**Comment by pirate486743186** [comment]
Yes, i read that later from the wiki. Actually all the information is already available in the wiki, but it's berried in it.

Something like an easily accessible getting started section is needed.

**Comment by Ghabry** [comment]
The main problem here is that we are all programmers and too lazy to update the website. We would need some highly motivated community manager like e.g. the Dolphin project has :/

**Comment by ItsSchoko** [comment]
This topic is somewhere in the back of my mind...

Plan is to tackle a lot of the stuff that comes up via Twitter etc. by creating pictured explanations & short clips that can be posted if redundant questions arise.
